trucks_project/api/main.py

```diff
@@ -1,63 +1,48 @@
-from flask import Flask
-from flask import jsonify
-from flask import request
-
-books = [
-    {
-        'id': 0,
-        'title': 'A Fire Upon the Deep',
-        'author': 'Vernor Vinge',
-        'first_sentence': 'The coldsleep itself was dreamless.',
-        'year_published': '1992'
-    },
-    {
-        'id': 1,
-        'title': 'The Ones Who Walk Away From Omelas',
-        'author': 'Ursula K. Le Guin',
-        'first_sentence': 'With a clamor of bells that set the swallows soaring, the Festival of Summer came to the city Omelas, bright-towered by the sea.',
-        'published': '1973'
-    },
-    {
-        'id': 2,
-        'title': 'Dhalgren',
-        'author': 'Samuel R. Delany',
-        'first_sentence': 'to wound the autumnal city.',
-        'published': '1975'
-    }
-]
+import json
+from flask import Flask, request, jsonify, abort
+from api.util.random import generate_random_uuid
+from truck_delivery_fernando.client_manager import ClientManager
 
-app = Flask(__name__)
+# from api.views import client_api
 
 API_NAME = "/api/v1"
+app = Flask(__name__)
 
 
-@app.route(f"{API_NAME}/books/<book_id>", methods=["GET"])
-def get_book(book_id):
-    #return jsonify(books[int(book_id)])
-    book_to_return = {}
-    for book in books:
-        if book.get("id") == int(book_id):
-            book_to_return = book
-            break
-    return jsonify(book_to_return)
-
+@app.route(f"{API_NAME}/client", methods=["POST"])
+def save_client():
+    client_params = json.dumps(request.json)
+    id = generate_random_uuid()
```

Comment:
if this is generated randomly how do you know which is the id of that client ? or is this just an example?

Reply:
The POST request returns which ID was generated by this method, we need to create unique objects when sending to the API.